api/go.mod

```diff
@@ -1,7 +1,7 @@
 module github.com/cilium/tetragon/api
 
 // renovate: datasource=golang-version depName=go
-go 1.21.6
+go 1.21.0
```

Comment:
Depending on the renovate configuration, it may end up trying to bump this again. In other projects we just configure renovate to only pick up updates for minor Go releases.

Reply:
we do that in Tetragon as well, here it was a deps updates that bumped it via `go mod tidy`

Reply:
https://github.com/cilium/tetragon/blob/5bf9880ea3672c2e95583ba91ce90d7d74fabdf9/.github/renovate.json5#L303-L326